deploy/default_values.yaml

```diff
@@ -62,6 +62,12 @@ spec:
   env:
     kubeEnv: okd
 
+  ###############################################################################
+  ## Logging configuration
+  ###############################################################################
+  logging:
```

Comment:
FYI: Without any code in the operator for both processing this default value and logging in general, this update doesn't have any effect.